File: C/pattern6.c
// # This is pentagon shape c-program
#include<stdio.h>
int main(){

    for (int i = 1;i<=11;i++){

    for (int j = 5;j>=i;j--){

        printf(" ");
    }

    for(int h = 1;h<=i;h++){  

        if (i<=5){
            printf("* ");
        }
        else{
            break;
        }
    }
    for (int o = 5;o>=i;o--){
        if (i>0){
            printf("* ");
        }
        else{
            break;
        }
    }
    for (int l = 6;l<=i;l++){
        printf(" ");
    }
    for (int k = 10;k>=i;k--){

        if (i>5){ 
            printf("* ");
        }
        else{   
            break;
        }

    }

    for (int s = 6;s<=i;s++){
        printf("* ");
    }

    for (int e = 1;e<=i;e++){

        if (i<=5){
            printf("* ");
        }
        else{
            break;
        }
    }
    
    for (int f = 10;f>=i;f--){ 

        if (i>5){

            printf("* ");
        }
        else{

            break;
        }

    }
    printf("\n");
    }

    return 0;
}

File: Python/pattern6.py
# This is pentagon shape python-program
for i in range(0,11,1):

    for j in range(5,i,-1):
        print(" ",end="")

    for h in range(0,i,1):
        if i<=5:
            print("*",end=" ")
        else:
            break

    for o in range(6,i,-1):
        if i>0:
            print("*",end=" ")
        else:
            break

    for l in range(6,i,1):
        print(" ",end="")
    
    for k in range(11,i,-1):
        if i>5:
            print("*",end=" ")
        else:
            break

    for s in range(5,i,1):
        print("*",end=" ")

    for e in range(0,i,1):
        if i<=5:
            print("*",end=" ")
        else:
            break
    
    for f in range(11,i,-1):
        if i>5:
            print("*",end=" ")
        else:
            break

    print("",end="\n")